### PR TITLE
fixed small issues

### PR DESCRIPTION
**Pull request by piotr-major-rmtcfm-com**

### CODE REVIEW

File: app/controllers/Auth/LoginController.js [hunk 1]
```diff
@@ -62,12 +60,12 @@ class LoginController {
         ) {
             return response
                 .status(StatusCodes.UNAUTHORIZED)
-                .send({ auth: false, token: null, user: null });
+                .send({ token: null, user: null });
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/controllers/Auth/LoginController.js]: also return only status

File: app/controllers/Auth/LoginController.js [hunk 2]
```diff
@@ -50,9 +50,7 @@ class LoginController {
         const user = await this.userRepository.getByEmail(email);
 
         if (!user) {
-            return response
-                .status(StatusCodes.UNAUTHORIZED)
-                .send({ auth: false, token: null });
+            return response.status(StatusCodes.UNAUTHORIZED);
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/controllers/Auth/LoginController.js]: sendStatus

File: app/controllers/Auth/ResetPasswordController.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
-//const ResetPassword = require('../../emails/ResetPassword');
 const { StatusCodes } = require('http-status-codes');
-const moment = require('moment');
+
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/controllers/Auth/ResetPasswordController.js]: remove this empty line

File: app/middleware/verifyToken.js [hunk 4]
```diff
@@ -12,12 +12,10 @@ module.exports = (request, response, next) => {
 
     jwt.verify(token, config.jwt.secret, (err, decoded) => {
         if (err) {
-            return response
-                .status(403)
-                .json({
-                    auth: false,
-                    message: 'Failed to authenticate token.'
-                });
+            return response.status(403).json({
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/middleware/verifyToken.js]: use http statuses pacakge

File: app/middleware/verifyToken.js [hunk 5]
```diff
@@ -12,12 +12,10 @@ module.exports = (request, response, next) => {
...
     jwt.verify(token, config.jwt.secret, (err, decoded) => {
         if (err) {
-            return response
-                .status(403)
-                .json({
-                    auth: false,
-                    message: 'Failed to authenticate token.'
-                });
+            return response.status(403).json({
+                auth: false,
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/middleware/verifyToken.js]: remove this `auth:false`

File: app/models/User.js [hunk 6]
```diff
@@ -1,6 +1,6 @@
 const { Model } = require('sequelize');
 const bcrypt = require('bcryptjs');
-const moment = require('moment');
+var dayjs = require('dayjs');
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/models/User.js]: const!

File: config/index.js [hunk 7]
```diff
@@ -19,7 +19,11 @@ module.exports = {
         logging: isEnabled(env('DATABASE_LOGGING')),
         host: env('DATABASE_HOST'),
         port: env('DATABASE_PORT'),
-        dialect: env('DATABASE_DIALECT')
+        dialect: env('DATABASE_DIALECT'),
+        define: {
+            charset: 'utf8mb4',
+            collate: 'utf8mb4_unicode_ci'
```
Comment (daniel-pupczyk-rmtcfm-com) [inline config/index.js:25]: Im not sure if it is enough, dont you need to add it also in migrations?

File: app/controllers/ToDo/StoreController.js [hunk 8]
```diff
@@ -50,12 +50,12 @@ class StoreController {
     }
 
     async invoke(request, response) {
-        const { body: data } = request;
+        const { loggedUserId, body: data } = request;
 
-        data.creatorId = request.loggedUserId;
+        data.creatorId = loggedUserId;
```
Comment (rafal-ksiazek-rmtcfm-com) [inline app/controllers/ToDo/StoreController.js]: TODO
There is better standard for "creatorId" => createdBy

File: app/middleware/verifyToken.js [hunk 9]
```diff
@@ -1,23 +1,22 @@
...
 const config = require('../../config');
 
 module.exports = (request, response, next) => {
     const token = request.token || null;
 
     if (!token) {
         return response
-            .status(403)
-            .json({ auth: false, message: 'No token provided.' });
+            .status(StatusCodes.NO_CONTENT)
```
Comment (rafal-ksiazek-rmtcfm-com) [inline app/middleware/verifyToken.js]: Why NO_CONTENT? 

without token - you are not authorized, so it should be 401. 

It's the same as you would not send sessionID.

File: app/middleware/verifyToken.js [hunk 10]
```diff
@@ -1,23 +1,22 @@
...
 
     jwt.verify(token, config.jwt.secret, (err, decoded) => {
         if (err) {
-            return response
-                .status(403)
-                .json({
-                    auth: false,
-                    message: 'Failed to authenticate token.'
-                });
+            return response.status(StatusCodes.NO_CONTENT).json({
```
Comment (rafal-ksiazek-rmtcfm-com) [inline app/middleware/verifyToken.js]: again, why NO_CONTENT?

File: app/routes/users.js [hunk 11]
```diff
@@ -1,6 +1,6 @@
 const express = require('express');
 
-const VerifyToken = require('../middleware/VerifyToken');
+const verifyToken = require('../middleware/verifyToken');
```
Comment (rafal-ksiazek-rmtcfm-com) [inline app/routes/users.js]: remove this empty line

we should use empty lines between groups of require
we have 2 groups: global libraries (like express) and our libraries (../middlewares ../services ../validators)

File: app/controllers/Auth/LoginController.js [hunk 12]
```diff
@@ -50,24 +50,20 @@ class LoginController {
...
 
         const userPassword = await this.userRepository.getPassword(user.id);
 
         if (
             !(await this.authService.comparePasswords(password, userPassword))
         ) {
-            return response
-                .status(StatusCodes.UNAUTHORIZED)
-                .send({ auth: false, token: null, user: null });
+            return response.status(StatusCodes.UNAUTHORIZED);
```
Comment (rafal-ksiazek-rmtcfm-com) [inline app/controllers/Auth/LoginController.js]: sendStatus

File: app/middleware/verifyToken.js [hunk 13]
```diff
@@ -1,23 +1,22 @@
...
 const config = require('../../config');
 
 module.exports = (request, response, next) => {
     const token = request.token || null;
 
     if (!token) {
         return response
-            .status(403)
-            .json({ auth: false, message: 'No token provided.' });
+            .status(StatusCodes.UNAUTHORIZED)
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/middleware/verifyToken.js:11]: there was 403 so FORBIDDEN, Im not sure which one should be there

File: app/middleware/verifyToken.js [hunk 14]
```diff
@@ -1,23 +1,22 @@
...
+            .json({ message: 'No token provided.' });
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/middleware/verifyToken.js]: .send

File: app/middleware/verifyToken.js [hunk 15]
```diff
@@ -1,23 +1,22 @@
...
 
     jwt.verify(token, config.jwt.secret, (err, decoded) => {
         if (err) {
-            return response
-                .status(403)
-                .json({
-                    auth: false,
-                    message: 'Failed to authenticate token.'
-                });
+            return response.status(StatusCodes.UNAUTHORIZED).json({
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/middleware/verifyToken.js]: as above

**Review by daniel-pupczyk-rmtcfm-com — APPROVED** [review]

File: app/middleware/verifyToken.js [hunk 16]
```diff
@@ -1,23 +1,22 @@
...
 
     if (!token) {
         return response
-            .status(403)
-            .json({ auth: false, message: 'No token provided.' });
+            .status(StatusCodes.UNAUTHORIZED)
+            .send({ message: 'No token provided.' });
     }
 
     jwt.verify(token, config.jwt.secret, (err, decoded) => {
```
Comment (rafal-ksiazek-rmtcfm-com) [inline app/middleware/verifyToken.js:15]: can we use "promisify" here?

File: app/controllers/Auth/ResetPasswordController.js [hunk 17]
```diff
@@ -1,6 +1,5 @@
-//const ResetPassword = require('../../emails/ResetPassword');
 const { StatusCodes } = require('http-status-codes');
-const moment = require('moment');
+const dayjs = require('dayjs');
 
 class ResetPasswordController {
     //PasswordResetController.js
```
Comment (rafal-ksiazek-rmtcfm-com) [inline app/controllers/Auth/ResetPasswordController.js:5]: do you need this?

File: app/middleware/verifyToken.js (hunk 13 above)
Comment (rafal-ksiazek-rmtcfm-com) [reply app/middleware/verifyToken.js:11]: if you want to go to endpoint you don't have permission but you are logged in then forbidden, if you are not logged in and you request for secured endpoint - then unauthorized

**Pull request merged**